site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -135,6 +135,21 @@ The name of the Application.
 An array of URLs that are the authorized origins for FusionAuth OAuth.
 +
 For improved security, all FusionAuth hosted login pages add an HTTP response header of `X-Frame-Options: DENY`. This response header disallows loading the FusionAuth pages from an iframe. To utilize an iframe and load one or more of the FusionAuth hosted login pages, add the iframe page URLs to this property. For that host, FusionAuth will remove the `X-Frame-Options` header allowing the page to load in the iframe.
++
+Examples of valid authorized origin URIs:
++
+* \https://example.com/example
+* com.myApp://example
+* com.myApp:/example
++
+[since]#Available since 1.32.0#
++
+You may now use URLs that do not begin with `http` to support native application origins. Prior to this version the value will be validated to begin with `http`.
++
+[since]#Available since 1.32.0#
++
+You may now use URLs for authorized origins that use a single slash to denote there is no naming authority for the scheme. Prior to this version a URL such as `com.myApp:/example` would fail validation as in invalid URL.
```

Comment:
Did you mean to enter this twice?

Reply:
No, but it did need to be re-worded. Was moving a bit too fast.  Should be fixed now.